docarray/array/storage/elastic/getsetdel.py

```diff
@@ -124,7 +124,7 @@ def _clear_storage(self):
 
     def _load_offset2ids(self):
         ids = self._get_offset2ids_meta()
```

Comment:
imo, also these functions should return something if `_enable_offset2id` is enable, otherwise return empty array